src/razor/src/dynamicFile/removeDynamicFileParams.ts

```diff
@@ -3,8 +3,9 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DocumentUri } from 'vscode-languageclient/node';
+import { TextDocumentIdentifier } from 'vscode-languageclient/node';
 
+// matches https://github.com/dotnet/roslyn/blob/9e91ca6590450e66e0041ee3135bbf044ac0687a/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/RazorDynamicFileInfoProvider.cs#L28
 export class RemoveDynamicFileParams {
-    constructor(public readonly razorFiles: DocumentUri[]) {}
+    constructor(public readonly csharpDocument: TextDocumentIdentifier) {}
```

Comment:
so ironically I named this wrong in the original PR 😅. I'll get that fixed later